.golangci.yml

```diff
@@ -90,13 +89,11 @@ linters:
     - noctx
     - rowserrcheck
     - staticcheck
-    - structcheck
     - stylecheck
     - typecheck
     - unconvert
     - unparam
     - unused
-    - varcheck
```

Comment:
do we want these golanci removals here?

Reply:
There were warnings about those linters being deprecated. This change comes from Ricardo, I could rollback it. No strong feelings, more than maybe checking for alternatives.

Reply:
Solved in fa509166f7920c636eaa5dff66d7494c40cb4155